index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">

```

Comment:
use indents. It's almost impossible to navigate in such code style. Your code have to look like this 
![image](https://user-images.githubusercontent.com/47325983/104041408-0b607500-51ea-11eb-8d02-984417167c01.png)